hoot-core-test/src/test/cpp/hoot/core/io/ServiceHootApiDbReaderTest.cpp

```diff
@@ -663,11 +662,9 @@ class ServiceHootApiDbReaderTest : public HootTestFixture
 
     OsmXmlWriter writer;
     writer.setIncludeCompatibilityTags(false);
-    writer.write(
-      map, "test-output/io/ServiceHootApiDbReaderTest/runReadByBoundsTestOutput.osm");
-    HOOT_STR_EQUALS(
-      TestUtils::readFile("test-files/io/ServiceHootApiDbReaderTest/runReadByBoundsTestOutput.osm"),
-      TestUtils::readFile("test-output/io/ServiceHootApiDbReaderTest/runReadByBoundsTestOutput.osm"));
+    writer.write(map, _outputPath + "runReadByBoundsTestOutput.osm");
+    HOOT_FILE_EQUALS( _inputPath + "runReadByBoundsTestOutput.osm",
```

Comment:
In this case HOOT_STR_EQUALS could be intentional since HOOT_FILE_EQUALS does a compareMaps() instead.